packages/search-in-workspace/src/browser/styles/index.css

```diff
@@ -224,6 +224,11 @@
     color: var(--theia-descriptionForeground);
     font-size: var(--theia-ui-font-size0);
     margin-left: 3px;
+    opacity: 0.7;
+}
+
+.theia-Tree .ReactVirtualized__List:focus .theia-TreeNode.theia-mod-selected .file-path {
```

Comment:
it should be scoped by `.t-siw-search-container`, not by generic tree or reactor selector, the same for other changes